meson.build

```diff
@@ -543,3 +544,28 @@ if get_option('gtest')
   ), args: '--gtest_output=xml:encoder.xml', timeout: 90)
 
 endif
+
+
+#############################################################################
+## Python bindings
+#############################################################################
+
+if get_option('python_bindings')
+  pymod = import('python')
+  python = pymod.find_installation('python3')
+  if python.language_version() < '3.7'
+    error('You need python 3.7 or newer')
+  endif  
+  py_bindings_generator = find_program('scripts/gen_py_bindings.py')
+
+  gen_py_bindings = custom_target('backends', input:[], output:['backends.cc'],
+    command : [py_bindings_generator, '@OUTPUT0@'])
+
+  py_files = [ gen_py_bindings ]
+
+  cpython = dependency('python3')
+  python.extension_module('backends',
+    [py_files + files],
+    include_directories: [includes],
```

Comment:
Adding `cpp_args: cc.get_supported_arguments(['-Wno-c99-extensions', '-Wno-pedantic', '-Wno-missing-field-initializers']),` here silences a lot of the warnings, but probably not a good idea.

Reply:
c99-extension that I use will appear in C++20, so I'd ignore that. Other ones probably worth fixing, but I'll do it later (clang also emits some set of warnings).